fluent-plugin-google-cloud.gemspec

```diff
@@ -20,12 +20,12 @@ eos
   gem.require_paths = ['lib']
 
   gem.add_runtime_dependency 'fluentd', '1.11.2'
-  gem.add_runtime_dependency 'googleapis-common-protos', '1.3.9'
-  gem.add_runtime_dependency 'googleauth', '0.9.0'
+  gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3.9'
+  gem.add_runtime_dependency 'googleauth', '~> 0.9'
   gem.add_runtime_dependency 'google-api-client', '0.30.8'
-  gem.add_runtime_dependency 'google-cloud-logging', '1.6.6'
-  gem.add_runtime_dependency 'google-protobuf', '3.12.2'
-  gem.add_runtime_dependency 'grpc', '1.31.1'
+  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.6'
```

Comment:
https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-logging/CHANGELOG.md#1109--2020-06-17

Reply:
This requires Ruby 2.4 now, as does googleauth v0.10.0 (https://github.com/googleapis/google-auth-library-ruby/blob/master/CHANGELOG.md#0100--2019-10-09). 

Isn't it time to drop support for Ruby 2.2 and 2.3?

Reply:
In any case, I reduced the version bumps and targeted the specific versions that were need to co-exist with `google-cloud-pubsub`.